jwst/lib/suffix.py

```diff
@@ -46,7 +46,7 @@
     'ramp', 'rate', 'rateints',
     's2d', 's3d', 'snr',
     'uncal',
-    'wfscmb', 'whtlt',
+    'wfscmb', 'whtlt','residual_fringe',
```

Comment:
Minor ADHD nit: Keeping this list alphabetical will help track down future issues 😄